LICENSE

```diff
@@ -178,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Diamond Light Source Ltd.
+   Copyright {yyyy} {name of copyright owner}
```

Comment:
Oopsie, @ptsOSL sorry I didn't review this PR earlier and catch this before it merged, but we can fix it on one of the open PRs

Reply:
If you read the preceding paragraph, it is intentional to keep the brackets and field description here.

Reply:
Ahh, my mistake! If I understand it correctly now, we are supposed to copy that line and the ones below into the header of every source file and switch out the bracketed bits when doing that, but leave them unchanged here?

Reply:
Technically, yes, but I'm not aware of people actually doing this properly...

Wait a bit and I'll see if something can be sorted out in the Copier template